docs/style-guide.md

```diff
@@ -319,6 +319,13 @@ This is 2 separate words. Not keyworker.
 
 The initialism LAU is probably not well-known enough yet to be used without explanation. Write local admin unit out in full the first time you use it. Use sentence case.
 
+{% endaccordionSection %}
+{% accordionSection "M" %}
+
+### MOJ
+
+Use MOJ for Ministry of Justice. This follows the [GOV.UK naming convention for government departments](https://www.gov.uk/guidance/style-guide/a-to-z#:~:text=Capitals%20for%20government%20departments).
```

Comment:
```suggestion
Use MOJ for Ministry of Justice. This follows the [GOV.UK naming convention for government departments](https://www.gov.uk/guidance/style-guide/a-to-z#capitals-for-government-departments).
```

I don't think we need the text-fragment search here `:~:` as I don't think it is supported in all browsers, and a simple heading id `#` seems to work to open the relevant accordion section.